### PR TITLE
[NUI] fix ScrollableBase wrong size issue

### PR DESCRIPTION
**Pull request by bshsqa**
- Fix OnMeasure callback in LinearLayout.(handling margin and padding)
 - Fix to handle only layout of ContentContainer to measure and layout of ScrollableBase.

### CODE REVIEW

File: src/Tizen.NUI/src/public/Layouting/LayoutItem.cs [hunk 1]
```diff
@@ -308,11 +308,11 @@ public void RequestLayout()
             Flags = Flags | LayoutFlags.ForceLayout;
             if (Parent != null)
             {
-                 LayoutGroup layoutGroup =  Parent as LayoutGroup;
-                 if(! layoutGroup.LayoutRequested)
-                 {
-                    layoutGroup.RequestLayout();
-                 }
+                LayoutGroup layoutGroup =  Parent as LayoutGroup;
+                if(! layoutGroup.LayoutRequested)
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/Layouting/LayoutItem.cs:312]: follow C# code format -> (Ctrl+K,F in Visual studio)

File: src/Tizen.NUI.Components/Controls/ScrollableBase.cs [hunk 2]
```diff
@@ -143,29 +131,27 @@ protected override void OnMeasure(MeasureSpecification widthMeasureSpec, Measure
                 widthSizeAndState.State = childWidthState;
                 heightSizeAndState.State = childHeightState;
 
-                SetMeasuredDimensions(ResolveSizeAndState(new LayoutLength(totalWidth + Padding.Start + Padding.End), widthMeasureSpec, childWidthState),
-                                       ResolveSizeAndState(new LayoutLength(totalHeight + Padding.Top + Padding.Bottom), heightMeasureSpec, childHeightState));
+                SetMeasuredDimensions(ResolveSizeAndState(new LayoutLength(totalWidth), widthMeasureSpec, childWidthState),
+                    ResolveSizeAndState(new LayoutLength(totalHeight), heightMeasureSpec, childHeightState));
```
Comment (huiyueun) [inline src/Tizen.NUI.Components/Controls/ScrollableBase.cs:135]: Remove space

Comment (bshsqa) [reply src/Tizen.NUI.Components/Controls/ScrollableBase.cs:135]: No matter

File: src/Tizen.NUI/src/public/Layouting/LayoutItem.cs (hunk 1 above)
Comment (bshsqa) [reply src/Tizen.NUI/src/public/Layouting/LayoutItem.cs:312]: Thanks I did.

**Comment by dasg34** [comment]
I approved this patch but only about `LinearLayout` side.
It will fix bug that `LinearLayout` padding is applied twice when cross-axis specification is `WrapContent`.

File: src/Tizen.NUI.Components/Controls/ScrollableBase.cs [hunk 3]
```diff
@@ -14,12 +14,13 @@
...
  */
 using System;
-using Tizen.NUI.BaseComponents;
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Diagnostics;
 using System.Runtime.InteropServices;
+
 using Tizen.NUI.Accessibility;
+using Tizen.NUI.BaseComponents;
```
Comment (hyunjushin) [inline src/Tizen.NUI.Components/Controls/ScrollableBase.cs:23]: are Classes in BaseComponents used in ScrollableBase?

**Review by hyunjushin — APPROVED** [review]

File: src/Tizen.NUI.Components/Controls/ScrollableBase.cs (hunk 3 above)
Comment (bshsqa) [reply src/Tizen.NUI.Components/Controls/ScrollableBase.cs:23]: View is in BaseComponents

**Pull request merged**